File: JavaScript/7kyu/Descending Order.js
function descendingOrder(n) {
  return parseInt(
    n
      .toString()
      .split("")
      .sort((a, b) => b - a)
      .join("")
  );
}

function descendingOrder_1(n) {
  return parseInt(String(n).split("").sort().reverse().join(""));
}

var result = descendingOrder_1(15);
console.log(result);

export { descendingOrder };

File: Python/7kyu/Descending_Order.py
def descending_order(num):
    lst = [n for n in str(num)]
    lst.sort(reverse=True)
    return int("".join(lst))


def descending_order_1(num):
    return int("".join(sorted(str(num), reverse=True)))


def descending_order_2(num):
    if isinstance(num, int) and num >= 0:
        return int("".join(sorted(str(num), reverse=True)))

    raise ValueError("Non-negative integer expected")


result = descending_order_2(46)
print(result)
